fix(docs): content review (#2131)
scaleway/docs-content: tutorials/create-openwrt-image-for-scaleway/index.mdx

For example:

> * In Scaleway Instances, the first NIC (e.g. `eth0`) must be associated to the public network and need to be configured (For example using: Cloud-init, DHCP, and others.)

Reviewer: ```suggestion
* In Scaleway Instances, the first NIC (e.g. `eth0`) must be associated to the public network and need to be configured (e.g. with Cloud-init, DHCP, or another method.)
```
